dashboard/app/helpers/application_helper.rb

```diff
@@ -70,8 +70,6 @@ def best_activity_css_class(user_levels)
       LEVEL_STATUS.review_rejected
     elsif result == Activity::REVIEW_ACCEPTED_RESULT
       LEVEL_STATUS.review_accepted
-    elsif user_level.try(:locked)
```

Comment:
once i updated `user_level` to have the virtual attribute `locked`, this started overriding the progress status of levels with `LEVEL_STATUS.locked`. i removed it, because theoretically this check previously never returned `true` unless there was an actual `locked` attribute at some point in the distant past.

Reply:
Good catch. The sort of thing that should have been removed when the attribute was removed...